### ISSUE TITLE
No Search results appearing

### ISSUE THREAD

**Issue by mannyy2020** [Ulauncher/Ulauncher#592]
### Steps to reproduce
1. Launch ULauncher
2. Start Typing
3. 

### Current behavior
 No search results appearing

### Expected behavior
Apps with similar names to appear

### Environment
**Ulauncher version**: 5.8.0
**Linux distro**: Ubuntu 20.04 LTS
**Desktop environment**: GNOME 3.36.3

### Logs
https://gist.github.com/mannyy2020/b747dcd57199b07a5a27c76ea2cfe8af.js

**Comment by noam09** [comment]
Hey, I'm not sure if this is relevant, but I was also getting a `JSONDecodeError` error, here's how you might approach debugging this. 
If you `ls -alt ~/.config/ulauncher/` you'll be able to see if any JSON file was modified recently. It's possible one of the JSON files Ulauncher uses (e.g. `shortcuts.json` or `extensions.json`) got corrupted or emptied. In my case, `shortcuts.json` was empty, so I removed it, which allowed Ulauncher to recreate it properly.

**TL;DR**
Most likely one or more of the JSON files in your config directory is malformed (Ulauncher expects it to look different), try to determine which and rename/remove it.

**Comment by NastuzziSamy** [comment]
Fixed as it was published
